bee-protocol/src/worker/milestone_validator.rs

```diff
@@ -141,6 +142,11 @@ where
     pub(crate) async fn run(mut self) -> Result<(), WorkerError> {
         info!("Running.");
 
+        async_std::task::spawn(async {
+            async_std::task::sleep(std::time::Duration::from_secs(30)).await;
+            Protocol::request_milestone_fill();
```

Comment:
So now the function `request_milestone_fill ` is called once and that's it ? Never again ?

Reply:
yeah I actually removed it at the end. We just request milestones as we need them. When they are confirmed, they get solidified.